enbeec/rgblent-client
# Swatch Component

## CHANGES:

- `src/hooks/useHover.js`
	- created `useHover` hook
	- returns an array containing a ref and the hover state of the ref'd component 
- `src/components/Swatch.js`
	- created `Swatch` component 
- `src/RGBlent.js`
	- added main view with two columns
		- left column
			- color picker 
			- left one swatch where the loaded color will go
		- right column
			- mock sidebar
- `src/index.js`
	- added a `BootstrapProvider`
	- added a simple (almost placeholder) theme
- new dependencies
	- now using `react-colorful` as a color picker

## TO TEST:

- Run the app with `npm start`
- Verify that you get a gray box on the right side of the app
- Verify that on the right you have:
	- a color picker up top
	- a circle underneath
		- upon hovering, there is a "hamburger" menu
		- updates color to when the color picker changes
	- both default to a nice pastel green
